connection_test.go

```diff
@@ -765,3 +765,33 @@ func TestConnectTimeout(t *testing.T) {
 		close(testComplete)
 	})
 }
+
+func TestTosPriority(t *testing.T) {
+	ctx, cancel := NewContext(time.Second)
+	defer cancel()
+
+	opts1 := testutils.NewOpts().SetServiceName("s1").SetTosPriority("LOWDELAY").NoRelay()
+	opts2 := testutils.NewOpts().SetServiceName("s2").SetTosPriority("LOWDELAY").NoRelay()
+	testutils.WithTestServer(t, opts1, func(ts *testutils.TestServer) {
+		ch2 := ts.NewServer(opts2)
+		hp2 := ch2.PeerInfo().HostPort
+		defer ch2.Close()
+
+		ts.Register(raw.Wrap(newTestHandler(t)), "echo")
+		ch2.Register(raw.Wrap(newTestHandler(t)), "echo")
+		outbound, err := ts.Server().BeginCall(ctx, hp2, "s2", "echo", nil)
```

Comment:
since the call is being made from s1 -> s2, I think s2 doesn't need the tos priority set right?